esp32-example.yaml

```diff
@@ -59,21 +59,37 @@ modbus_controller:
 
 binary_sensor:
   #  11  Status/Fault flag                     2 byte   R  uint16  Hex See ^3
-  #      Bit 8: Status: Charging enabled/disabled
+  #      Bit 8: Status: Charging operation
+  - platform: modbus_controller
+    modbus_controller_id: bms0
+    name: "${name} charging operation"
```

Comment:
Could you describe the meaning of this binary sensor in a few words? I would like to try to improve the naming of these 4 status sensors to be more explicit.

Reply:
I'm doing some experiments with the PACE BMS to elucidate the behavior, it's new to me.

Reply:
Alright! Let me know as soon you know the differences between the flags.

Reply:
OK some additional experience with PACE BMS:
If  over voltage protection is reached, the SOC is set to 100% and the remaining capacity is set to design capacity.
After arrive of overvoltage protection release voltage, the specified protection flag is cleared, but not Bit16!  
As long as the protection-flag bitset is not equal 0, the balancing works also (Bit16 is on). 
Bit16 is cleared, if the Mosfet-switch get state enabled.